lib/content-services/src/lib/document-list/components/document-list.component.ts

```diff
@@ -894,7 +896,7 @@ export class DocumentListComponent implements OnInit, OnChanges, OnDestroy, Afte
         this._pagination.maxItems = requestPaginationModel.maxItems;
         this._pagination.merge = requestPaginationModel.merge;
         this._pagination.skipCount = requestPaginationModel.skipCount;
-        this.reload();
+        this._pagination.merge ? this.reloadWithoutResettingSelection() : this.reload();
```

Comment:
reload() {
        this.ngZone.run(() => {
            this.resetSelection();
              if(this._pagination.merge){
                          this.reloadWithoutResettingSelection();
              }
        });
    }

Reply:
to me looks better leave all thsi responsibility in the reload method